tests/search/nearest_neighbor/test.sd

```diff
@@ -24,14 +21,13 @@ search test {
       expression: sqrt(sum(join(query(qpos_float), attribute(pos), f(x,y)((x-y)*(x-y)))))
     }
     first-phase {
-      expression: 15 - itemRawScore(nns)
+      expression: 15 - distance(nns)
     }
     summary-features {
       euclidean_distance_qpos_double
       euclidean_distance_qpos_float
-      rawScore(pos)
```

Comment:
Perhaps we should continue testing rawScore and itemRawScore until they get different behavior?

Reply:
i'll test them again after changing their values (which will happen soon)

Reply:
Ok, sounds good